pootle/static/js/editor/app.js

```diff
@@ -2312,4 +2338,40 @@ PTL.editor = {
     return true;
   },
 
+  toggleSuggestion(e, canHide = false) {
+    e.stopPropagation();
+    const $suggestion = $(e.target).parents('.js-user-suggestion');
+
+    if (this.selectedSuggestionId === undefined) {
+      this.selectedSuggestionId = $suggestion.data('sugg-id');
+      const props = {
+        editor: this,
+        suggId: this.selectedSuggestionId,
+        suggText: $suggestion.data('translation-aid'),
+      };
+      const $feedbackPlaceHolder = $suggestion.find('.feedback')[0];
+      $suggestion.addClass('suggestion-expanded');
+      $('.js-editor-body .translate-full').addClass('suggestion-expanded');
+      this.suggestionFeedbackForm = ReactDOM.render(
+        <SuggestionFeedbackForm {...props} />,
+        $feedbackPlaceHolder
+      );
+      $('#suggestion-editor').focus();
```

Comment:
This is `SuggestionFeedbackForm`'s responsibility. You need to hook into the [`componentDidMount()` lifecycle method](https://facebook.github.io/react/docs/component-specs.html#mounting-componentdidmount), retrieve the reference to the DOM node, and call `focus()` there. You can check the [lifecycle method in the `SearchBox` component](https://github.com/translate/pootle/blob/5f49268035438e30c650c0c66758ed216534379a/pootle/static/js/admin/components/SearchBox.js#L36-L38) for an example.